### DISCUSSION TITLE
Offset format in default configuration

### DISCUSSION THREAD

**Discussion by Tuuuukka** [dunst-project/dunst#1400] [General]
Shouldn't offset be in format `offset = 10x50` instead of what's in the default `dunstrc`:

https://github.com/dunst-project/dunst/blob/423c3a1a1a9ec96cb9d7a9e76d641f12c7e5eada/dunstrc#L39-L40

At least for me this gives an error, where as `10x50` works as expected..?

**Comment by bynect** [comment]
This behavior has been changed in the master. I should add some reference to the change in the dunstrc sorry

**Reply by Tuuuukka** [reply]
No worries, good to know 👍🏻

**Comment by bynect** [comment]
I added a note to the offset in dunstrc. could you check if you think it is clear enough?

**Reply by Tuuuukka** [reply]
Looks good to me, thanks!
